rotkehlchen/api/rest.py

```diff
@@ -1270,15 +1266,6 @@ def query_list_of_all_assets(
                 entries_table='assets',
             )
 
-        if identifiers is not None and len(identifiers) != assets_found:
-            not_found = []
-            found_assets = {asset['identifier'] for asset in assets}
-            for asset in identifiers:
-                if asset not in found_assets:
-                    not_found.append(asset)
-            msg = f"Queried identifiers {','.join(not_found)} are not present in the database"
-            return api_response(wrap_in_fail_result(msg), status_code=HTTPStatus.CONFLICT)
```

Comment:
then you need to update the docs but why this change?

Reply:
This part of the code looked a bit strange to me. I asked Luki and he said that this check is not used / relied upon anywhere anyway, so I removed the unused code.